zerobuf/Zerobuf.cpp

```diff
@@ -41,19 +41,29 @@ void Zerobuf::setZerobufData( const void* data, size_t size )
 std::string Zerobuf::toJSON() const
 {
     Json::Value rootJSON;
-    for( const auto& valueSchema : getSchema().fields )
-        addValueToJSON( rootJSON, valueSchema )
+    for( auto valueSchema : getSchema().fields )
```

Comment:
Still a copy then. If there is no real reason to change that, keep const ref please.